.github/workflows/test.yml

```
@@ -29,42 +29,29 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version:
-        - "3.9"
-        - "3.10"
-        - "3.11"
-        - "3.12"
-        - "3.13"
-        os:
-        - "windows-latest"
-        - "macos-latest"
-        - "ubuntu-latest"
-        # Python 3.9 is on macos-13 but not macos-latest (macos-14-arm64)
-        # https://github.com/actions/setup-python/issues/696#issuecomment-1637587760
-        exclude:
-        - { python-version: "3.9", os: "macos-latest" }
-        include:
-        - { python-version: "3.9", os: "macos-13" }
-
+        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13"]
+        os: [windows-latest, macos-latest, ubuntu-latest]
```

The current line-style version minimises git diffs, which is nicer
```suggestion
        - "3.9"
        - "3.10"
        - "3.11"
        - "3.12"
        - "3.13"
        os:
        - "windows-latest"
        - "macos-latest"
        - "ubuntu-latest"
```

---

I think this change is nicer, and consistent with other repos. OS values hardly ever change, and version not very often.

But I'm not too bothered so we can maintain the line-style :)